CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+## [42.2.9] (2019-12-03)
+## [42.2.9] (2019-12-04)
+### Changed
+
+### Added
+
```

Comment:
`- Add support for large update counts (JDBC 4.2 compatibility) [PR 935](https://github.com/pgjdbc/pgjdbc/pull/935)`

Reply:
seems all of the changes are missing as well as 42.2.8

Reply:
Actually the 42.2.8 does have a "Fixed" item, but since Changed and Added are empty, they should be removed there...

I don't recall the process completely, but `CHANGELOG.md` should be filled **before** executing `release_notes.sh`.

The intent is to fill `CHANGELOG.md` with all the relevant and/or highlighted items, this should _ideally_ be done in the respective PR adding a human-readable item in the `CHANGELOG.md` reducing the load to maintainers (you) of remember what features put there, for every "important" PR it should be asked to add the entry in the `CHANGELOG.md`, the potential issue with that is merge conflicts... but when all the items are in place in the `CHANGELOG.md`, the `release_notes.sh` is run to generate the respective pages properly filled.